README.md

```diff
@@ -64,7 +64,7 @@ There are multiple ways to provide these credentials:
 
 | Option | Description                                                                                              |
 |--------|----------------------------------------------------------------------------------------------------------|
-| **1**  | Set an environment variable explicitly: `AICORE_SERVICE_KEY`                                             |
+| **1**  | Set an `.env` file containing an `AICORE_SERVICE_KEY`                                                    |
```

Comment:
(Major/Question)

This would be an addition right? Not a replacement?

Reply:
(Minor)

Maybe making it more explicit?
```suggestion
| **1**  | Set an `.env` file containing an `AICORE_SERVICE_KEY={...}`                                             |
```

Reply:
The "expected state" of the ticket indicates something else.

Reply:
Updated to accept both